core/src/main/kotlin/br/com/concrete/yosef/api/property/spacing/PaddingPropertyCommand.kt

```diff
@@ -22,12 +22,12 @@ class PaddingPropertyCommand : DynamicPropertyCommand {
 
     override fun apply(view: View, dynamicProperty: DynamicProperty) {
 
-        val (left, top, right, bottom) = generateMarginList(dynamicProperty, view)
+        val (left, top, right, bottom) = generatePaddingList(dynamicProperty, view)
```

Comment:
Same here about checking the four values

Reply:
done